libs/core/hardware/CMakeLists.txt

```diff
@@ -24,6 +25,7 @@ set(hardware_compat_headers
     hpx/util/hardware/timestamp/linux_generic.hpp => hpx/modules/hardware.hpp
     hpx/util/hardware/timestamp/linux_x86_32.hpp => hpx/modules/hardware.hpp
     hpx/util/hardware/timestamp/linux_x86_64.hpp => hpx/modules/hardware.hpp
+    hpx/util/hardware/timestamp/linux_riscv_64.hpp => hpx/modules/hardware.hpp
```

Comment:
This is a new file, we don't need any compatibility header for it. This line is not needed.

Reply:
Cool. Will remove it this evening.